tokio-postgres/tests/test/main.rs

```diff
@@ -147,6 +147,58 @@ async fn scram_password_ok() {
     connect("user=scram_user password=password dbname=postgres").await;
 }
 
+#[tokio::test]
+async fn host_only_ok() {
+    let _ = tokio_postgres::connect(
```

Comment:
@sfackler I am not sure if these tests should belong here since we only test `connect_raw` here. But I think these tests do have some values.